Makefile

```diff
@@ -2,11 +2,11 @@
 SHELL = /bin/bash
 .SHELLFLAGS += -e
 
-KERNEL_ABI_MINOR_VERSION = 2
-KVERSION_SHORT ?= 4.9.0-9-$(KERNEL_ABI_MINOR_VERSION)
+KERNEL_ABI_MINOR_VERSION = 11-2
```

Comment:
>KERNEL_ABI_MINOR_VERSION [](start = 0, length = 24)

No need to change KERNEL_ABI_MINOR_VERSION, it is still 2.

Reply:
Done.
What about debian/config/defines, we still have abiname=11-2
The default abiname for debian 4.9.189 is 11, and we want a different abiname (such as 11-2)
If this is the same ABI, the script debian/bin/buildcheck.py will fail to verify the ABI